corehq/apps/users/views/mobile/users.py

```diff
@@ -1430,10 +1431,15 @@ def count_users(request, domain, include_mobile_users=False, include_web_users=F
         user_filters = form.cleaned_data
     else:
         return HttpResponseBadRequest("Invalid Request")
+
     user_count = 0
-    for domain in user_filters['domains']:
-        user_count += get_users_by_filters(domain, user_filters, include_mobile_users=include_mobile_users,
+    (is_multi_domain_download, domains_list) = get_domains_from_user_filters(domain, user_filters)
+    for current_domain in domains_list:
+        user_count += get_users_by_filters(current_domain, user_filters, include_mobile_users=include_mobile_users,
                                            include_web_users=include_web_users, count_only=True)
+        if include_web_users:
+            user_count += Invitation.by_domain(current_domain).count()
```

Comment:
Invitations are purposefully part of the single-domain download, which was built for USH, and the multi-domain download should be consistent with the single-domain download.

Users will show up in the multi-domain download once for each domain they belong to - in code terms, this is really a domain membership download. The columns are about 50/50 user columns vs membership columns: name & email will be repeated for users who belong to multiple domains, but role, last access/login, and location are all domain-specific.

Reply:
> Invitations are purposefully part of the single-domain download, which was built for USH

Okay. But its a part of the usual product now. I suspect it would be confusing for others but most of them won't have invites to cause that confusion, and also something product then decided to go with.

> the multi-domain download should be consistent with the single-domain download

That makes sense.

Just a curious follow up,
What happens if someone edits the invitation in the downloaded sheet and uploads it back?